pkg/jiraservicedesk/client/client_test.go

```diff
@@ -0,0 +1,78 @@
+package client
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/nbio/st"
+	"gopkg.in/h2non/gock.v1"
+)
+
+var endpoint = "177"
+var client = NewClient("", "https://mock.atlassian.net/", "")
+var jira_url = "https://mock.atlassian.net/rest/api/3"
+
+var project = Project{
+	Name:               "Stakater",
+	Key:                "STK",
+	ProjectTypeKey:     "service_desk",
+	ProjectTemplateKey: "com.atlassian.servicedesk:itil-v2-service-desk-project",
+	Description:        "Sample project for jira-service-desk-operator",
+	AssigneeType:       "PROJECT_LEAD",
+	LeadAccountId:      "5ebfbc3wwe226gfda32c3590",
+	URL:                "https://stakater.com",
+}
+
+func TestJiraServiceDesk_DeleteProject_withValidProjectId_shouldDeleteProject(t *testing.T) {
```

Comment:
Let's try to be consistent in following naming conventions for test cases. For reference: https://github.com/stakater/slack-operator/blob/master/pkg/slack/service_test.go#L13